packages/ember-headless-form/src/components/headless-form.gts

```diff
@@ -1,12 +1,13 @@
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { assert, warn } from '@ember/debug';
-import { hash } from '@ember/helper';
+// @ts-ignore hash is missing from ember types
```

Comment:
But linting was happy with this file. IIRC `modifier` was actually missing, not? But somehow it was not needed, the template was still working for me locally without the import 🤔

Reply:
yeah, it's a built-in, but not yet for types -- I think there is an open issue about it and it came up in discord a couple days ago